interface/modules/custom_modules/oe-module-visualehr/Readme.md

```diff
@@ -0,0 +1,26 @@
+## Visual EHR
+This is a brief readme of how to install the visual dashboard.
+
+This is a reactjs application embeded in a php module, hence this installation is in two steps
+
+1. ReactJs Configuration
+STEP 1: 
+In the react directory (oe-module-visualehr/react), headover to the package.json file and change the homepage and proxy values to the settings that applies to you. for example
+`"homepage": "http://localhost/openemr-7/interface/modules/custom_modules/oe-module-visualehr/public/"`,
```

Comment:
Does react not give you any way to use a relative path? I worry this will generate a lot of support requests if everyone has to customize this on the first step.  Also perhaps these instructions should go into the Modules -> Configure page if you want people to know how to use this.  As I checked out the branch and the first thing I got was a 404 white screen.

Reply:
package.json is actually in oe-module-visualehr/react/vehr/

Reply:
Didn't realize openemr-7 was in the url.  Recommend removing this from your sample app and go with what comes bundled into OpenEMR by default.